src/xERC20/contracts/XERC20Lockbox.sol

```diff
@@ -145,4 +146,19 @@ contract XERC20Lockbox is Initializable, IXERC20Lockbox {
     receive() external payable {
         depositNative();
     }
+
+    /**
+     * @notice Redeem ERC20 tokens by the XERC20 contract
+     *
+     * @param _to The address to send the tokens to
+     * @param _amount The amount of tokens to redeem
+     */
+    function redeemByXERC20(address _to, uint256 _amount) external {
```

Comment:
can we rename is to `redeemERC20onL1`? maybe event have a check that chainid = 1? because this should not be possible on L2s

Reply:
`redeemERC20ByXERC20onL1`

Reply:
keeping it as it is.